app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -50,6 +50,24 @@ export const PressGallery: React.FC = () => {
     [findContentWithElasticsearch],
   );
 
+  /** separate requests to find total hits for each press member */
+  const fetchResultHits = React.useCallback(
+    async (filter: MsearchMultisearchBody, name?: string, date?: string) => {
+      try {
+        const res: any = await findContentWithElasticsearch(filter, false);
```

Comment:
Why `any`?  Our results from Elasticsearch should now be strongly typed.  See examples already created.